docusaurus/docs/iOS/customization/localization.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Translation and i10n
```

Comment:
Change the title to: Adding Localization

Reply:
Changed.